src/python/turicreate/test/test_image_classifier.py

```diff
@@ -264,17 +264,8 @@ def test_export_coreml_predict(self):
                     self.tolerance,
                 )
         else:
-            deep_features = data[0:1][self.feature][0]
-            if _mac_ver() >= (10, 13):
-                classes = self.model.classifier.classes
-                ret = coreml_model.predict({self.feature: deep_features})
-                coreml_values = [ret[self.target + "Probability"][l] for l in classes]
-
-                self.assertListAlmostEquals(
-                    coreml_values,
-                    list(self.model.predict(deep_features, output_type="probability_vector")),
-                    self.tolerance,
-                )                
+            # If the code came here that means the type of the feature used is deep_deatures and the predict fwature in coremltools doesn't work with deep_features yet so we will ignore this specific test case unitl the same is written.
```

Comment:
Should "fwature" be "function"? We try to maintain a 80 or 100 character length line limit. I would be nice if the text wrapped to the next line after 80 or 100 characters.